shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -696,16 +703,18 @@ public void process(RequestProcessor cp) {
         @Override
         public void writeFields(DataOutput out) throws IOException {
             out.writeLong(requestId);
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
```

Comment:
And here. And everywhere else. 
Please consider bumping the supported versions to make any previous client/server versions incompatible

Reply:
Yes. We need to bump up the protocol version.